routers/web/auth/saml.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2024 The Gitea Authors. All rights reserved.
```
There are more files with 2023.